### DISCUSSION TITLE
404 for approved and reviewed Dapp

### DISCUSSION THREAD

**Discussion by JumpTaskDev** [WalletConnect/walletconnect-monorepo#2824] [Developer Support] [answered]
Hi,

our project has recently been reviewed and approved to be listed on the Explorer. We can see the project in the list, but [the inner page](https://walletconnect.com/explorer/jumptask) returns 404.

Could you please help with this issue?

**Comment by crypblizz8** [comment]
Investigating this for you now @JumpTaskDev 

Does it block you from anything else in terms of development?

**Comment by chadyj** [comment, marked as answer]
Fixed!
https://walletconnect.com/explorer/jumptask

Thanks for reporting.
